scholly/base/src/global/fakeData.js

```diff
@@ -2,8 +2,8 @@ export const FAKE_TEAM_SITE_METADATA = {
   pages: ['PUBLICATIONS', 'TEAM', 'ACHIEVEMENTS'],
   publicationOptions: { layout: 'All Publication', sortBy: 'Author' },
   template: {
-    layout: '2',
-    theme: 'light',
+    layout: '3',
+    theme: 'dark',
```

Comment:
We probably want to keep the old defaults as its local testing. I'd say change up the configuration as needed locally, but not to commit the changes.

Reply:
Done